README.md

```diff
@@ -21,12 +21,35 @@ Feature planning is done on [our Trello board](https://trello.com/b/STeIZ1td/pro
 
 ## How to Develop
 
-You will need Ruby and Rubygems installed.
+You will need Ruby, Rubygems and PostgreSQL installed.
 
 ```bash
 bundle install
-bin/rake db:migrate db:seed
+bin/rake db:setup
```

Comment:
Is this all that's necessary? I've run `createdb` commands locally to set up dev and test databases for projects, but maybe I was doing that unnecessarily.